app/queries/servers/thread.ts

```diff
@@ -112,7 +113,7 @@ export const prepareThreadsFromReceivedPosts = async (operator: ServerDataOperat
     let processedThreads: Set<string> | undefined;
 
     posts.forEach((post: Post) => {
-        if (!post.root_id && post.type === '') {
+        if (!post.root_id && ['', PostTypes.CUSTOM_CALLS].includes(post.type)) {
```

Comment:
This took awhile to track down -- we weren't treating call posts as thread-able. Please let me know if there's someplace I've missed.